src/test/java/com/systelab/seed/patient/PatientResourceTest.java

```diff
@@ -118,8 +157,8 @@ private void createSomePatients(int numberOfPatients) {
                     .when().post("/patients/patient")
                     .then().assertThat().statusCode(200)
                     .extract().as(Patient.class);
-            //This check is identical for all patients and shall be specific or merged in one informing about creation of {0} patients.
-            TestUtil.checkObjectIsNotNull("Patient ID " + patientCreated.getId(), patientCreated.getId());
+
+            Assertions.assertNotNull(patientCreated.getId(), "Patient not created");
```

Comment:
The intention of the comment you just deleted was to have an specific expected result for each patient created (using ID or something shorter like patient 1, patient 2,...patient n, OR use a unique expected result for the whole batch of patients created like "5 patients successfully created".
Now, I can not understand your "Patient not created" message.

Reply:
It was agreed with Olga that some verifications or checks in the returned values of auxiliar functions would not be reported to allure (as they are reported in other tests), so I replaced the TestUtil.checkObjectIsNotNull for the Assertions.assertNotNull. The verification is equally done in both cases. But the Assertion does not report anything to allure, and just displays the 'Patient not created' message when the assertion fails, that is, when the patients has not been created, and the deletion cannot be verified.

Reply:
"Patient not created" is the message that will be displayed in case the assertion is not null

Reply:
Ok, got it. The declaration in TestUtils confused me (even worse, this new assertion does not use TestUtils..).